components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```diff
@@ -185,26 +187,30 @@ def main(argv):
     comp_jobs_dir = clp_config.logs_directory / "comp-jobs"
     comp_jobs_dir.mkdir(parents=True, exist_ok=True)
 
-    if parsed_args.input_list is None:
-        # Write paths to file
-        log_list_path = comp_jobs_dir / f"{str(uuid.uuid4())}.txt"
-        with open(log_list_path, "w") as f:
-            for path in parsed_args.paths:
+    paths_to_compress = []
+    if len(compress_paths_arg) > 0:
+        for path in compress_paths_arg:
+            stripped_path = path.strip()
+            if "" == stripped_path:
+                # Skip empty paths
+                continue
+            resolved_path_str = str(pathlib.Path(stripped_path).resolve())
+            paths_to_compress.append(resolved_path_str)
+    else:
+        # Read paths from the input file
+        log_paths_to_compress = pathlib.Path(compress_path_list_arg).resolve()
```

Comment:
`compress_path_list_path`? `log_paths_to_compress` sounds like a list but in this case it's a file path to a list of log paths.